File: dev-docs/snippets/collection-type-configs/get.swift
import JsBaoClient

// Fetch the configuration for one collection type. Returns a typed
// `CollectionTypeConfigInfo`; `ruleSetId` is an optional `String`.
func get(client: JsBaoClient, collectionType: String) async throws {
  // #region example
  let config = try await client.collectionTypeConfigs.get(collectionType: collectionType)
  let ruleSetId = config.ruleSetId
  // #endregion example
  _ = (config, ruleSetId)
}

File: dev-docs/snippets/collection-type-configs/get.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Fetch the configuration for one collection type.
export async function get(client: JsBaoClient, collectionType: string) {
  // #region example
  const config = await client.collectionTypeConfigs.get(collectionType);
  // #endregion example
  return config;
}
